package.json

```diff
@@ -18,6 +18,7 @@
 	"files": [
 		"index.js"
 	],
+	"typings": "index.d.ts",
```

Comment:
There's is no need to add this since it's named `index`, but you do need to add it to the `files` property.